pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -273,9 +273,37 @@ func ContainerFromNative(n *native.Container) (*Container, error) {
 		}
 		c.NetworkSettings = nSettings
 	}
+
+	c.Config = &Config{
+		Hostname: n.Labels[labels.Hostname],
+		Labels: filterLabels(n.Labels,
+			hasPrefix(labels.Prefix),
```

Comment:
trying to hide some labels used by nerdctl or added by containerd

https://github.com/containerd/nerdctl/pull/1260/files#r929190294